test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -50,10 +50,12 @@ public HubConnectionTests(ServerFixture<Startup> serverFixture, ITestOutputHelpe
             using (StartLog(out var loggerFactory, $"{nameof(CheckFixedMessage)}_{protocol.Name}_{transportType}_{path.TrimStart('/')}"))
             {
                 var connection = new HubConnectionBuilder()
-                    .WithUrl(_serverFixture.Url + path)
-                    .WithTransport(transportType)
                     .WithLoggerFactory(loggerFactory)
                     .WithHubProtocol(protocol)
+                    .WithHttpConnection(_serverFixture.Url + path, options =>
```

Comment:
I would make the transport type a first class as well argument as well.

Reply:
Optional argument? Currently it default to All.

Reply:
Yea

Reply:
Got me thinking but is off topic - https://github.com/aspnet/SignalR/issues/1842